src/coreComponents/constitutive/fluid/multifluid/CO2Brine/functions/FlashModelBase.hpp

```diff
@@ -101,6 +112,31 @@ class FlashModelBase
    */
   virtual void checkTablesParameters( real64 pressure, real64 temperature ) const = 0;
 
+  /**
+   * @brief Print the table(s) in the log and/or CSV files when requested by the user.
+   * @param tableData The target table to be printed
+   * @param pvtOutputOpts Struct containing output options
+   */
+  void outputPVTTableData( TableFunction const * tableData, TableOutputOptions const pvtOutputOpts )
```

Comment:
Also, does this method should be a member function? Could it be static or a free function?